docs/user-guide/debugging-gso.md

```diff
@@ -13,4 +13,6 @@ let io = s2n_quic::provider::io::Default::builder()
 let mut server = Server::builder()
     .with_io(io)?
     .start()?;
-```
\ No newline at end of file
+```
+
+In the case of running tests in broken GSO environments, setting `S2N_QUIC_PLATFORM_FEATURES_OVERRIDE=""` as an environment variable will disable the feature.
```

Comment:
I'm not sure we want to document this right now. It's a bit of a blunt instrument and will disable _all_ feature probing. I think we could improve usability, where you can specify something like `S2N_QUIC_PLATFORM_FEATURES_OVERRIDE=default,-gso,-gro`

Reply:
Sorry I wasn't clear - the `-gso` isn't implemented today 😅 so we should just remove this paragraph and probably open an issue to say we should implement and document it